.github/workflows/question-service.yml

```diff
@@ -8,7 +8,7 @@ on:
   pull_request:
 
 jobs:
-  Test:
```

Comment:
Curious what's the issue here? I don't think it will stop working if it is named "Test"

Reply:
Oh previously I added another job so I rename the job name, but after that I realized the newly added job is not working so I remove the job but forgot to change back the name